File: PHP/car.php
<?php
require_once('account.php');
class Car{
    public $id;
    public $license;
    public $driver;
    protected $passanger;

    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }
    
    public function PrintDataCar(){
        echo   "license: $this->license
                Driver: {$this->driver->name}
                Document: {$this->driver->document}
                Number of passengers: $this->passanger";
    }

    public function getPassengers(){
        return $this->passanger;
    }

    public function setPassenger($passenger) {
        if ($passenger <= 4) {
            $this->passanger = $passenger;
        }
        else {
            echo "Please asing no more than 4 passengers";
        }
    }
}
?>

File: Java/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    protected Integer passenger;
    
    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("License: " + license);
        System.out.println("Driver: " + driver.name);
        System.out.println("Passengers Allowed: " + passenger);
    }

    public Integer getPassenger(){
        return passenger;
    }

    public void setPassenger(Integer passenger){
        if (passenger >= 4) {
            this.passenger = passenger;
        }
        else{
            System.out.println("Please set a number equal or higer than 4");
        }
    }
}
